### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Dadadon**
- Added styling to project using bootstrap

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,20 +1,56 @@
...
+        {" "}
+        <div style={{ width: 75, marginLeft: 20 }}>
+          <CircularProgressbar value={percentage} />
+        </div>
+        <div className="mt-3 ms-2">
+          <p className="mb-0">{`${percentage}%`}</p>
+          <p className="text-muted">Completed</p>
+        </div>
+        <div class="Line-2"></div>
+      </div>
```
Comment (roshan-bajgain) [inline src/components/Book.js]: - [ ] You should stick to the design as much as possible like Font, color, and spacing. There is a logo at the top of the page. please implement it. Please use the same font and color that is in the design.
### Your design
your
### Figma design
fimga

File: src/components/AddBook.js [hunk 2]
```diff
@@ -18,25 +18,29 @@ const AddBook = () => {
...
           id="author"
           type="text"
           name="author"
           required
           placeholder="Author"
         />
-        <button type="submit" >Add Book</button>
+        <button type="submit" className='btn btn-sm btn-primary' >Add Book</button>
       </form>
-    </>
```
Comment (roshan-bajgain) [inline src/components/AddBook.js]: - [ ] You should stick to the design as much as possible that is color, font, and spacing. please adjust the spacing between the input field and use the same color in the Figma design
### Your design
yourrrrrrrrrrrr
### Figma design
figmaaaaaaaaaaa

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Dadadon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Dadadon, 

Good job implementing the changes requested by the previous reviewer

To highlight 👍🏽 
- You added the person icon
- You used the right colors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**